.yarnclean

```diff
@@ -0,0 +1,4 @@
+# unwanted packages
+node_modules/@types/graphql
+node_modules/apollo-link-context/node_modules/@types/graphql
+node_modules/apollo-link-http-common/node_modules/@types/graphql
```

Comment:
This will make yarn delete these obsolete type packages on install, to ensure we never get mixed types because that’s a horrible experience.